.clang-format

```diff
@@ -0,0 +1,96 @@
+---
+Language:        Cpp
```

Comment:
why not cpp11 or 17?

Reply:
None, Cpp, CSharp, Java, JavaScript, ObjC, Proto, TableGen, TextProto are the only available options for this field.